src/api/procedures/inviteAccount.ts

```diff
@@ -66,15 +67,6 @@ export async function prepareInviteAccount(
     });
   }
 
-  const isPresent = !!secondaryKeys.find(({ signer }) => signerToString(signer) === address);
-
-  if (isPresent) {
-    throw new PolymeshError({
-      code: ErrorCode.ValidationError,
-      message: 'The target Account is already a secondary key for this Identity',
-    });
-  }
-
```

Comment:
Why did you remove this validation?

Reply:
Ah ok. This error can not be thrown because if the key has any Identity, the previous error would throw first.